src/main/java/com/squareup/tools/maven/resolution/ProxyHelper.kt

```diff
@@ -21,6 +21,7 @@ private val URL_PATTERN =
   Pattern.compile("^(https?)://(([^:@]+?)(?::([^@]+?))?@)?([^:]+)(?::(\\d+))?/?$")
 
 object ProxyHelper {
+  private val okHttpClient: OkHttpClient = OkHttpClient()
```

Comment:
maybe call it clientTemplate or rootClient, since that's how we're using it.

Reply:
changed to rootClient!